react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx

```diff
@@ -67,22 +71,42 @@ function InviteByEmailSection({ inviteSubject, inviteText, inviteTextiOS }: IPro
     const dispatch = useDispatch();
     const { classes } = useStyles();
     const { t } = useTranslation();
+    const [ isClicked, setIsClicked ] = useState(false);
     const encodedInviteSubject = encodeURIComponent(inviteSubject);
     const encodedInviteText = encodeURIComponent(inviteText);
     const encodedInviteTextiOS = encodeURIComponent(inviteTextiOS);
 
     const encodedDefaultEmailText = isIosMobileBrowser() ? encodedInviteTextiOS : encodedInviteText;
 
+    useEffect(() => {
+        mounted = true;
+
+        return () => {
+            mounted = false;
+        };
+    }, []);
+
     /**
      * Copies the conference invitation to the clipboard.
      *
      * @returns {void}
      */
-    function _onCopyText() {
+    async function _onCopyText() {
         dispatch(showSuccessNotification({
             titleKey: 'dialog.copied'
         }, NOTIFICATION_TIMEOUT_TYPE.SHORT));
-        copyText(inviteText);
+        const isCopied = await copyText(inviteText);
```

Comment:
I would drop the async/await - think it's safe to assume the copy is almost instant and always successful.

Reply:
But for this we will remove the condition also

Reply:
yes please disregard this :) it's ok to leave isCopied, I see we have precedence as well on other components